logbook-spring/src/test/java/org/zalando/logbook/spring/RemoteResponseTest.java

```diff
@@ -7,19 +7,9 @@
 import java.io.IOException;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
 
 class RemoteResponseTest {
 
-    @Test
-    void statusCanThrow() throws IOException {
```

Comment:
Any particular reason why this test was removed?

Reply:
It was removed due to the dependency upgrade. I have reverted this commit at https://github.com/zalando/logbook/pull/1304/commits/2a167380774f04f47b6cbded8dbbb934d360d1a6 to see if it can pass the build.